src/models/reposyncstate.ts

```diff
@@ -215,15 +215,26 @@ export class RepoSyncState extends Model implements RepoSyncStateProperties {
 		return results[0] as RepoSyncState & Subscription;
 	}
 
-	static async findAllFromSubscription(subscription: Subscription, options: FindOptions = {}): Promise<RepoSyncState[]> {
+	static async findAllFromSubscription(subscription: Subscription, limit: number, offset: number, order: Order, options: FindOptions = {}): Promise<RepoSyncState[]> {
 		const result = await RepoSyncState.findAll(merge(options, {
 			where: {
 				subscriptionId: subscription.id
-			}
+			},
+			limit,
+			offset,
+			order
 		}));
 		return result || [];
 	}
 
+	static async countAllFromSubscription(subscription: Subscription, options: FindOptions = {}): Promise<number> {
```

Comment:
This function looks to do the same thing as countSubscriptionRepos on line 182

Reply:
Awesome find! :killwithfire: